packages/components/icons/src/icons/Check.tsx

```diff
@@ -0,0 +1,12 @@
+export const Check = () => (
+  <svg
+    viewBox="0 0 24 24"
+    xmlns="http://www.w3.org/2000/svg"
+    data-title="check"
+    fill="currentColor"
+    stroke="currentColor"
+  >
+    <title>check</title>
```

Comment:
👀

Reply:
this is 4 a11y purposes

Reply:
https://testing-library.com/docs/queries/bytitle/